app/src/main/java/com/alexbar/layoutbasic/ContentScreen.kt

```diff
@@ -1,50 +1,60 @@
 package com.alexbar.layoutbasic
 
+// Compose Foundation Layouts
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
+
+// Compose Material3 Components
 import androidx.compose.material3.Surface
+
+// Compose Runtime
 import androidx.compose.runtime.Composable
+
+// Compose UI
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.painterResource
-import androidx.compose.ui.unit.dp
+
+// Project-Specific Imports
 import com.alexbar.layoutbasic.ui.theme.BackgroundColor
+import com.alexbar.layoutbasic.ui.theme.Dimens
+import com.alexbar.layoutbasic.utils.MusicConstants
```

Comment:
@ezatpanah When you said "Group your imports for better organization" did you mean like this?